Sources/MapboxMaps/Foundation/Camera/CameraAnimationsManager.swift

```diff
@@ -15,8 +15,17 @@ internal protocol CameraAnimatorInterface: CameraAnimator {
     func update()
 }
 
+internal protocol CameraAnimationsManagerProtocol: AnyObject {
+    func ease(to camera: CameraOptions,
```

Comment:
what is the advantage of this being protocoled? Can we just have the methods in the `CameraAnimationsManager`

Reply:
The gesture handlers depend on this protocol instead of depending on `CameraAnimationsManager` directly so that we can inject mocks when we test them.

Reply:
That makes sense, so it sounds like it is solely for testing purposes as opposed to SDK functionality/architecture?

Reply:
Yep, it's just an implementation detail for testing.